test/controllers/api/v2/host_collections_controller_test.rb

```diff
@@ -67,6 +62,8 @@ def test_create
                     :host_collection => {:name => 'Collection A', :description => 'Collection A, World Cup 2014',
                                          :system_ids => [@system.id]}
 
+      assert_response :success
```

Comment:
i moved this above the other assertions as it will typically 'fail' before the others and is most important